blog/_posts/2017-09-04-opensource-here-we-come.md

```diff
@@ -0,0 +1,87 @@
+
+He is a junior at IIITD, majoring in Computer science, Polygot Programmer, passionate about Opensource, with a side of research. Shiven is a 2016 GSoC intern with Fossasia and worked in Loklak.
+
+#### [Aditay Tripathi](https://github.com/Aditay/)
+
+He has just completed his MTech from our college only and did his BTech from NIT Shrinagar. 
+He is an AI enthusiast in Research and applications of Machine learning/deep learning in speech recognition and computer vision. He is well versed with various libraries like Kaldi, Tensorflow, Theano and Keras.
+
+### Our Mentors
+![image](/img/blog/2017/Gemini_mentors.jpg)
+<div align="center" class="image-credits">Our Mentors(Photo: Github)</div>
+
+#### [Stefan van der Walt](https://github.com/stefanv)
+He is an assistant researcher at BIDS and a senior lecturer in applied mathematics at Stellenbosch University, South Africa. He has been involved in the development of scientific open source software for more than a decade and enjoys teaching Python at workshops and conferences.  Stéfan is the founder of scikit-image and a contributor to numpy, scipy, and dipy.
+
+#### [Juan Nunez-Iglesias](https://github.com/jni)
+He is a research scientist at the University of Melbourne, with interests in neuroscience and biological image analysis. He is also a freelance consultant in scientific Python programming, with a particular interest in renewable energy and the environment.Also, he is the author of the book Elegant SciPy, published by O'Reilly media.
+
+A big thank you to our new mentors for taking us on board at such a short notice!
+
+### Supervisor 
+
+#### [Vaishali](https://github.com/nerdyvaishali)
+
+She is an Open Source enthusiast and works as a Linux kernel engineer at Oracle. She is also a blogger and outreachy alumni. She motivates everyone to get started with linux kernel and also takes up workshops for the same.
+
+# Our Project(s)
+
+## About Susi-AI Server
+[Susi AI](http://chat.susi.ai/overview) is an open source artificial Intelligence for chatbots, personal assistants, help desks. Recognising the need to personalise chatbots Susi also provides a platform to personalise skills to a user’s requirements.
+## About Scikit-image
+[Scikit-image](http://scikit-image.org/) is an open source image processing library for the Python which includes collection of algorithms for segmentation,color space manipulation, analysis, filtering etc. It's largely written in Python, with some core algorithms written in Cython to achieve performance. Scikit-image is designed to interoperate with the Python numerical and scientific libraries NumPy and SciPy.
+
+## What we’ve done so far and some unexpected events :O 
+
+Time indeed flies. We’ve already sifted through the large code base, understood it, made small PRs here and there, learnt git, understood what a frontend architecture like react does and realized the effort a community puts in maintaining a project. 
```

Comment:
Post looks better to me now. Only one concern here. It's not clear about which project you are talking here. It's totally fine to mention which project was assigned to you previously, what you learnt/worked on and which project you will be working on now. Can you please add project names and things accordingly? Thanks.